src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -121,26 +121,25 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                 ColorConsole.WriteLine();
                 foreach (var line in lines)
                 {
-                    ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
+                    ColorConsole.WriteLine(new ColorToken[]
+                    {
+                        padding
+                    }.Concat(line).ToArray());
                 }
 
                 ColorConsole.WriteLine(
                     padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")} to {currentLeaderboard.LastActivityDate?.ToLocalTime().ToString() ?? ("started")}".Gray());
 
                 var maxMessageLength = 0;
                 var refreshTime = DateTime.UtcNow.AddMilliseconds(refreshInterval);
                 using (var timer = new Timer(c =>
                 {
-                    var timeLeft = new TimeSpan(0, 0, 0, (int)Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
-                    var message = $"\r{padding}{DateTime.UtcNow.ToLocalTime()} · Refreshing in {timeLeft.Humanize()}...";
-                    maxMessageLength = Math.Max(maxMessageLength, message.Length);
-                    ColorConsole.Write(message.PadRight(maxMessageLength).DarkGray());
+                    var timeLeft = new TimeSpan(0, 0, 0, (int) Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
+                    var totalMessage = $"Total {itemsName}: {currentLeaderboard?.Count ?? 0:N0}"; 
+                    var refreshingMessage = $" · Refreshing in {timeLeft.Humanize()}...";
+                    maxMessageLength = Math.Max(maxMessageLength, totalMessage.Length + refreshingMessage.Length);
+                    ColorConsole.Write("\r", padding, totalMessage.Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]), refreshingMessage.PadRight(maxMessageLength).DarkGray());

```

Comment:
The padding is there because the length of the message changes depending on how many digits there are in the number of seconds left, e.g. when the `refreshInterval` is 10, the initial message says "10 seconds" but the subsequent one says "9 seconds". The padding pads out the message to the max length seen. (Note that `refreshInterval` comes from config.)

As for the colouring, I'd *prefer* to keep it restricted to just the number. It fits better with the rest of the board, e.g. when an entry moves up in the ranking, only the number gets colour bright green.